devicetypes/sinope-technologies/dm2550zb-sinope-dimmer.src/dm2550zb-sinope-dimmer.groovy

```diff
@@ -0,0 +1,336 @@
+/**
+Copyright Sinopé Technologies
+1.3.0
+SVN-571
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+**/
+
+preferences {
+	input("MinimalIntensityParam", "number", title:"Light bulb minimal intensity (1..10) (default: blank)", range:"1..10", description:"optional")
+    // when the is at a low value, some bulbs may flicker for some technical reasons. to prevent that behaviour. writting this parameter will increase the minimal value
+    // of the dimmer's become a little bit higher so the load doesn't start flickering when the level is low.
+
+    input("LedIntensityParam", "number", title:"Indicator light intensity (1..100) (default: blank)", range:"1..100", description:"optional")
+	input("trace", "bool", title: "Trace", description: "Set it to true to enable tracing")
+	// input("logFilter", "number", title: "Trace level", range: "1..5",
+	// 	description: "1= ERROR only, 2= <1+WARNING>, 3= <2+INFO>, 4= <3+DEBUG>, 5= <4+TRACE>")
+}
+
+metadata {
+    definition (name: "DM2550ZB Sinope Dimmer", namespace: "Sinope Technologies", author: "Sinope Technologies",  ocfDeviceType: "oic.d.switch")
```

Comment:
@aguilbault-Sinope the intention was to integrate this device in an existing DTH, so this one (DM2550ZB Sinope Dimmer) is not needed.  The same fingerprint cannot be in many DTHs.
If there is any configuration needed for Your device - please add proper code to zigbee-dimmer.groovy DTH.

Reply:
@KKlimczukS it's my first integration so i'm kinda new to this, my apologies. Could I  instead then simply add the fingerprint to one of our previously created DTH that is doing the same thing ? file dm2500zb-sinope.dimmer.groovy ?

Reply:
@KKlimczukS I went with your first option of implementing the device into the existing DTH (zigbee-dimmer.groovy). Let me know if there is something I did wrong.

Also, I guess I'll have to delete the existing DTH in the smartthings developper console/Groovy IDE. Is there something else that needs to be done ?

Thanks a lot for your help!

Reply:
@aguilbault-Sinope I took a look at Your changes in zigbee-dimmer.groovy. 
My intention was to ask You to change absolutely minimum (like configuration etc), You added some additional functions and since this DTH `runs locally` then not all Your code will work there.

I suggest to try to integrate it in dm2500zb-sinope.dimmer.groovy and check if it's working as expected.
To make it possible to test on Your account (groovy IDE - graphapi) - You need at least to comment out the fingerprint in the DTH that will not be used with this device.